### DISCUSSION TITLE
Adjust iPad Sidecar Display Brightness Using Built-In MacBook Brightness Keys?

### DISCUSSION THREAD

**Discussion by jacobjivanov** [MonitorControl/MonitorControl#921] [Q&A]
![image](https://user-images.githubusercontent.com/59805681/151687081-5658a518-0fac-41ed-bcb1-72fb63fbf60d.png)
While I really appreciate the consolidation of brightness controls across my MacBook and iPad, and soon to be third monitor, I would like the to control it using the built-in MacBook brightness keys. Is this possible at the moment, or is it planned to be in the future?

I really like the app btw. I would contribute but my Swift skills are just... not there

**Comment by waydabber** [comment]
Hi, you can adjust the sidecar display brightness using software adjustment (the hardware brightness of the iPad is unaffected).

On Monterey 12.3 and Universal Control you can change the brightness of the iPad using the brightness keys natively in Universal Control mode (but not in Sidecar mode).

**Reply by jacobjivanov** [reply]
Since it is possible to change the iPad display brightness when in Sidecar by moving the mouse cursor onto its display, is there a way you can "sync" them automatically?

**Reply by waydabber** [reply]
Hi @jacobjivanov - you can set up both your keyboard and the sliders to affect all displays at once. 

<img width="658" alt="Screen Shot 2022-01-31 at 18 17 03" src="https://user-images.githubusercontent.com/37590873/151841234-d5df8b51-4ab4-42c9-83d3-58503ff1ebed.png">

<img width="634" alt="Screen Shot 2022-01-31 at 18 17 21" src="https://user-images.githubusercontent.com/37590873/151841274-c68ad891-d3f6-4a1f-a341-fad283064525.png">
